core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -436,12 +436,23 @@ oppia.directive('conversationSkin', [
             }
 
             if (ExplorationPlayerStateService.isStateTerminal(stateName)) {
-              ExplorationRecommendationsService.getRecommendedSummaryDicts(
-                ExplorationPlayerStateService.getAuthorRecommendedExpIds(
-                  stateName),
-                function(summaries) {
-                  $scope.recommendedExplorationSummaries = summaries;
-                });
+              var parentExplorationId = UrlService.getParentExplorationId();
+              if (parentExplorationId) {
+                $scope.parameters = UrlService.getParameters();
```

Comment:
$scope.parentExplorationIds = UrlService.getParentExplorationIds();

(and maybe, for clarity, rename the method above to UrlService.getImmediateParentExplorationId() or getFirstParentExplorationId() or getLastParentExplorationId(), whichever makes sense)

Reply:
Done